src/RouteJs/RouteJs.cs

```diff
@@ -35,6 +42,21 @@ IEnumerable<IConstraintsProcessor> constraintsProcessors
 		}
 
 		/// <summary>
+		/// Gets or sets whether urls are converted to lower case.
+		/// </summary>
+		public static bool LowerCaseUrls { get; set; }
```

Comment:
I don't think this option should be needed, you can just use the option on `RouteCollection` itself.

Also, please don't add any static configuration to this file. It's intentionally using dependency injection to ensure there's no global state where possible.